ichnaea/data/tests/test_validation.py

```diff
@@ -45,18 +48,6 @@ def check_normalized_cell(self, measure, cell, expect):
                         actual=result[k]))
         return result
 
-    def check_normalized_wifi(self, measure, wifi, expect):
-        d = measure.copy()
-        d.update(wifi)
-        result = normalized_wifi_measure_dict(d)
-
-        if expect is None:
-            self.assertEqual(result, expect)
-        else:
-            for (k, v) in expect.items():
-                self.assertEqual(result[k], v)
-        return result
-
     def make_cell_submission(self, **kw):
         measure = dict(radio='umts',
```

Comment:
Actually a top-level radio entry of umts wouldn't be valid. Top level is meant to be the phone type, which can only be '', 'gsm' or 'cdma'.